taskcluster/ci/signing-bundle/kind.yml

```diff
@@ -19,9 +19,16 @@ job-template:
         kind: build
         symbol: S-AAB
         tier: 2
-    # TODO Use "signing" once Nightly AABs have their own production key
-    worker-type: dep-signing
+    worker-type:
+        by-variant:
+            nightly: signing
+            default: dep-signing
     worker:
         max-run-time: 3600
-        # TODO Use "release-signing" once Nightly AABs have their own production key
```

Comment:
Mmmm, I don't recall why I suggested we use `release-signing` instead of `nightly-signing` in https://github.com/mozilla-mobile/reference-browser/pull/901. Maybe it's because there's a single app on the Play Store (unlike Fenix, but like Focus). In any case, it's not a blocker.

Reply:
Yes, it is a little odd. But that's also the way nightly apk signing is set up -- at least it is consistent in that sense.